vite_ruby/lib/vite_ruby/package_manager.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ViteRuby::PackageManager
+  def self.resolve(root:)
+    package_manager_name = ENV.fetch('VITE_RUBY_PACKAGE_MANAGER', detect_package_manager(root))
+    package_manager_class_for(package_manager_name).new(root:)
```

Comment:
@pftg Missing the actual reference to the `root` variable. This is what's breaking the builds.
Should be:
```rb
package_manager_class_for(package_manager_name).new(root: root)
```

Reply:
Okay - managed to pull down the project and fix the error which then lead to more build errors. Not sure where to go from here

Reply:
> @pftg Missing the actual reference to the `root` variable. This is what's breaking the builds. Should be:
> 
> ```ruby
> package_manager_class_for(package_manager_name).new(root: root)
> ```

Oh, I forgot about other ruby versions. You are right

Reply:
Hey @schonert, how can I trigger running all checks?